conjure/snip/imports.go

```diff
@@ -21,8 +21,30 @@ import (
 const (
 	pal = "github.com/palantir/"
 	cgr = pal + "conjure-go-runtime/v2/"
+	wgs = pal + "witchcraft-go-server/v2/"
 )
 
+var DefaultImportsToPackageNames = map[string]string{
```

Comment:
I don't like having to maintain this list (even though it only has to match the other entries in this file). Open to suggestions on how to build this mapping more dynamically but since all the imports are global variables I don't think there's a way to iterate over the full list and pull out the imports.

Reply:
I think this is reasonable enough and at least provides a central place to keep things updated. An alternative would be to inspect the package graph and pull out the imports used by a specific project, but either way we need a sane mapping to package names which I don't think we can avoid here.